src/main/java/org/mondemand/Client.java

```diff
@@ -71,6 +69,7 @@
   private ConcurrentHashMap<EventType, List<Transport>> transports = null;
   private ClientStatEmitter autoStatEmitter = null;
   private Thread emitterThread = null;
+  private Integer maxNumMetrics = null;
```

Comment:
So I changed things around a little bit.  maxNumMetrics is here so it can be passed to Transports via the send() call.  Transports can be added after a Client has been constructed, so it seems like it makes sense for Client to be passing this parameter during send()